build.rs

```diff
@@ -85,51 +77,6 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
         blindbid_keys.update("blindbid", blindbid::compile_circuit()?)?;
     }
 
-    // Get the cached keys for transfer contract crate from rusk profile, or
```

Comment:
I'm talking about the blindbid circuits, not the  transfer ones whose keys are computed inside the transfer-circuits crate

Reply:
I have everything locally. Unfortunately, we need to make complex changes to the build script to make it build. The transfer contract update will reimplement this part of the build script and it was removed. However, there should be no changes to the blindbid part

Reply:
I double checked, and there is no change in the blindbid part. The removed part is related to the transfer circuit only, because it was entirely recreated